test/e2e/framework/service_util.go

```diff
@@ -490,13 +501,18 @@ func (j *ServiceTestJig) newRCTemplate(namespace string) *v1.ReplicationControll
 			Labels:    j.Labels,
 		},
 		Spec: v1.ReplicationControllerSpec{
-			Replicas: func(i int) *int32 { x := int32(i); return &x }(1),
+			Replicas: func(i int) *int32 { x := int32(i); return &x }(2),
```

Comment:
... not your fault, but could use just `var replicas  int32 = 2` at the top of this function and use `&replicas` here.